docs/guide/tft_bestpractices.md

```diff
@@ -22,7 +22,7 @@ and the TensorFlow
 [Keras](https://www.tensorflow.org/guide/keras/overview) API.
 
 The second document,
-[Data preprocessing with Google Cloud](../tutorials/transform/data-preprocessing-with-cloud),
+[Data preprocessing with Google Cloud](../tutorials/transform/data_preprocessing_with_cloud),
```

Comment:
Would you please change this into `data_preprocessing_with_cloud.md`?

Reply:
of course, just to double check, if I added the `.md` ext won't this cause the online doc to refer to https://www.tensorflow.org/tfx/tutorials/transform/data_preprocessing_with_cloud.md?